action.yml

```diff
@@ -8,12 +8,12 @@ inputs:
   personalAccessToken:
     required: true
     description: The Azure DevOps Personal Access Token (PAT) that has permission to publish symbols
-  symbolsFolder: 
+  symbolsFolder:
     required: false
     description: The base directory where the source code exists (defaults to GITHUB_WORKSPACE)
-  searchPattern: 
+  searchPattern:
     required: false
-    description: The search pattern used to search for symbols
+    description: The search patterns used to search for symbols, separated by newlines.
```

Comment:
yaml has the notion of a list. I think it would be more idiomatic to support either a string value or a list of values, rather than a multi-line string containing \n's. That's a pretty common way to evolve a json or yaml schema. Did you consider this?

Reply:
I don't believe GitHub actions allows lists/arrays as action input types. You can use `fromJson` to parse a JSON string containing a `[ 'list', 'of', 'items' ]` but I figured that was less convenient than newlines. I could be wrong! It's difficult to find official documentation on action inputs.  The _workflow_ inputs are documented [here](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype) and only support string, number, or boolean.

Reply:
ah if arrays are not natively supported then your solution makes sense